### DISCUSSION TITLE
HAR version is not supported. Currently supporting:[1.1, 1.2]

### DISCUSSION THREAD

**Discussion by lalithaawgu** [microcks/microcks#1394] [Q&A] [answered]
When importing https://raw.githubusercontent.com/twilio/twilio-oai/main/spec/json/twilio_api_v2010.json for mocks, I see this error `HAR version is not supported. Currently supporting: [1.1, 1.2]`
Apicurio is able to import it fine however. Am i missing any json property ?

**Comment by lbroudoux** [comment, marked as answer]
Hey there!

It should work OOTB, but it appears that for unknown reasons (at the moment 😉), the file is detected as HAR (HTTP Archive) instead of OpenAPI v3. We have to work on this. 

Would you mind creating an issue on https://github.com/microcks/microcks/issues?

**Reply by lbroudoux** [reply]
Issue #1394 has been opened to track the bug and it is now fixed in `nightly` version.
